v5/api/c/motors.rst

```diff
@@ -116,7 +116,7 @@ Analogous to `pros::Motor::move_absolute <../cpp/motors.html#move-absolute>`_.
 ============ ===============================================================
  port         The V5 port number from 1-21
  position     The absolute position to move to in the motor's encoder units
- velocity     The maximum allowable velocity for the movement
+ velocity     The maximum allowable velocity for the movement in RPM

```

Comment:
Is this really RPM or is it the motor's encoder units?

Reply:
https://github.com/purduesigbots/pros/blob/ab99a3391c8d88129e479ff6ce450afe7e569d10/include/pros/motors.h#L78

Reply:
Yeah, I found out last season that it's always in RPM regardless of the motor's encoder units. Seemed a bit strange but held up in my testing.